lib/MicrophoneSettings.tsx

```diff
@@ -9,6 +9,8 @@ export function MicrophoneSettings() {
   const { isNoiseFilterEnabled, setNoiseFilterEnabled, isNoiseFilterPending } = useKrispNoiseFilter(
     {
       filterOptions: {
+        bufferOverflowMs: 100,
+        bufferDropMs: 200,
```

Comment:
trying to avoid out-of-sync scenarios that feel like >2s out of sync.
In theory it should only be a max of 500ms before the buffer gets dropped, but in testing it seemed longer than that.